spec/components/candidate_interface/course_choices_review_component_spec.rb

```diff
@@ -353,6 +353,28 @@
     end
   end
 
+  context 'when an interview has been scheduled' do
+    it 'renders the component with interview details' do
+      application_choice = create(:application_choice, :with_completed_application_form, :with_scheduled_interview)
+      application_form = application_choice.application_form
```

Comment:
```suggestion
    let(:application_choice) { create(:application_choice, :with_completed_application_form, :with_scheduled_interview) }
    let(:application_form) { application_choice.application_form }
    
    it 'renders the component without interview details' do    
```

Reply:
@despo we've followed the 'let's not' approach on the project (https://thoughtbot.com/blog/lets-not). I must admit I've used `let` everywhere on all the other projects I've worked on so I'm happy either way - but for consistency we should probably stick with the explicit local variables. What do you think?

Reply:
I have not seen this article before. However I'm not sure I agree with it. I find that in a lot of our specs it's very hard to understand what's going on because of the amount of setup that is included in them.

I understand how it can be an issue if all the data setup happens at the beginning of the spec, but doing that within the context block means that they are easy to see, understand and keep up to date if the spec changes in the future. Also, I'm not sure I get this point `It causes Slow Tests by creating more data than is necessary in each test.` as anything defined with let is not evaluated until the first time it's called.